vignettes/use_tutoriales.Rmd

```diff
@@ -0,0 +1,35 @@
+---
+title: "use_tutoriales"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{use_tutoriales}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Partes del tutorial 
+
+Todos los tutoriales incluidos tienen una forma similar a la siguiente:
+
+![](img/forma_general.png)
```

Comment:
I moved img to vignettes/img. I assume img was only needed in vignettes. But please check.